src/listfunc.c

```diff
@@ -1692,32 +1693,32 @@ Obj FuncCOPY_LIST_ENTRIES( Obj self, Obj args )
   else
     {
       Obj tmplist = NEW_PLIST(T_PLIST,number);
-      Obj *tptr = ADDR_OBJ(tmplist)+1;
-      sptr = ADDR_OBJ(srclist)+srcstart;
+      sptr = CONST_ADDR_OBJ(srclist)+srcstart;
+      dptr = ADDR_OBJ(tmplist)+1;
       for (ct = 0; ct < number; ct++)
         {
-          *tptr = *sptr;
-          tptr++;
+          *dptr = *sptr;
+          dptr++;
           sptr += srcinc;
         }
-      tptr = ADDR_OBJ(tmplist)+1;
```

Comment:
Ahhh... I guess the "problem" was that `tptr` is once used for writing (above), and then again for reading (starting at this line), so with different const needs.
But since we had it for writing anyway, it doesn't matter so much that we later loop over it for reading, at least for the purpose of sneaking in read/write guards later on. So, yeah, I should have left `tptr` alone :/